### ISSUE TITLE
Did you manage to reproduce the results from the original paper?

### ISSUE THREAD

**Issue by dvornikita** [Jongchan/tensorflow-vdsr#1]
In my case it gives the results not better then bicubic interpolation, although I might do something wrong.

**Comment by Jongchan** [comment]
I tried many setups for training, by changing dataset/learning rate/gradient clipping.
The best result is always 0.1~0.3 psnr lower than the original paper.
I forgot the best setup, but as soon as I get the result, I will share it here.

**Comment by hetaoaoao** [comment]
Same for me... it doesn't reproduce the paper result,  psnr are nearly the same with bicubic resize.

**Comment by Jongchan** [comment]
@hetaoaoao There are several reasons for that. 

1. gradient variation is too high
2. training data is too many
3. learning rate is too high (this is usually not the case)

Therefore, I recommend you to use a lower gradient clipping norm, or less training examples.

I should have preserved the best setup.. but I finished this code several months ago, somehow the best results are gone. Sorry for any confusions made by me.

I will share the best model and the hyper parameter setup, ASAP.

**Comment by dvornikita** [comment]
@Jongchan Thank you very much.

**Comment by Jongchan** [comment]
@dvornikita  @hetaoaoao 
Currently, I've managed to achieve the result below. (not fully trained, just a 45-epoch result)
Please allow 2-3 more days for better results, I hope I can achieve somewhat better results and faster training.

##### Results on Set 5

|  Scale    | Bicubic | VDSR | tf_VDSR |
|:---------:|:-------:|:----:|:-------:|
| **2x** - PSNR/SSIM|   33.66/0.9929	|   37.53/0.9587	|  37.16 |
| **3x** - PSNR/SSIM|   30.39/0.8682	|   33.66/0.9213	|  33.32 |
| **4x** - PSNR/SSIM|   28.42/0.8104	|   31.35/0.8838	|  31.03 |

##### Results on Set 14

|  Scale    | Bicubic | VDSR | tf_VDSR |
|:---------:|:-------:|:----:|:-------:|
| **2x** - PSNR/SSIM|   30.24/0.8688	|   33.03/0.9124	| 32.81 |
| **3x** - PSNR/SSIM|   27.55/0.7742	|   29.77/0.8314	| 29.62 |
| **4x** - PSNR/SSIM|   26.00/0.7027	|   28.01/0.7674	| 27.83 |

**Comment by Jongchan** [comment]
I'd like to further improve the result, but will close this issue for now.
Link to the actual dataset / latest PSNR result / latest training code is updated.
If I have any further updates, will push to the git.
